subprojects/kotlinTest/kotlinTest.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.1.4-3'
+    // Can not use libraries.kotlin.version here per https://stackoverflow.com/questions/37555196/in-gradle-how-to-use-a-variable-for-a-plugin-version
+    id 'org.jetbrains.kotlin.jvm' version '1.1.60'
```

Comment:
We could probably wrap the duplication if we used traditional way of declaring plugins, via "buildscript" section. See the main "build.gradle" file if you are interested. I think it is a good idea to avoid duplication but I'm not too concerned.